api/healthcheck/healthcheck_controller.go

```diff
@@ -0,0 +1,76 @@
+/*
+ *    Copyright 2018 The Service Manager Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package healthcheck
+
+import (
+	"net/http"
+
+	"github.com/Peripli/service-manager/pkg/web"
+	"github.com/Peripli/service-manager/rest"
+	"github.com/Peripli/service-manager/storage"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	apiVersion = "v1"
+	root       = "monitor/health"
+	// URL is the path of the healthcheck endpoint
+	URL = "/" + apiVersion + "/" + root
```

Comment:
Can we make URL private?

Reply:
It is used in the `healthcheck_test.go:54`